xayagametest/testcase.py

```diff
@@ -83,6 +85,14 @@ def main (self):
 
     randomSuffix = "%08x" % random.getrandbits (32)
     self.basedir = os.path.join (self.args.dir, DIR_PREFIX + randomSuffix)
+    
+    if sysconfig.get_platform() == "mingw":
+      self.args.dir = tempfile.gettempdir()
+      self.basedir = os.path.join (self.args.dir, DIR_PREFIX + randomSuffix)
```

Comment:
I believe the proper way to fix this is updating how the default argument is set in the parser, around here: https://github.com/xaya/libxayagame/blob/74ae5848d1b4620c0d9ea658ba4dc5269f00369b/xayagametest/testcase.py#L58

Probably just removing `DEFAULT_DIR` and instead using `gettempdir()` on every platform is what we should do.